vulture_os/toolkit/api_parser/harfanglab/harfanglab.py

```diff
@@ -160,8 +161,32 @@ def get_logs(self, since=None, to=None, index=0):
                      extra={'frontend': str(self.frontend)})
         return self.__execute_query("GET", alert_url, payload)
 
-    def format_log(self, log):
+    def get_threats(self, since=None, to=None, index=0):
+        threat_url = f"{self.harfanglab_host}/{self.THREATS}"
+        # Remove miliseconds - format not supported
+        since_utc = since.isoformat().split('.')[0]
+        to_utc = to.isoformat().split('.')[0]
```

Comment:
don't use methods on objects that are defined as `None` by default without checking their type first!
also, you might prefer using `strftime()` if you need to generate a custom string format